### PR TITLE
GDEV-184 Create AWS Workspace Optimization Module

### PR DESCRIPTION
**Pull request by TysonBeck**

### CODE REVIEW

**Review by jeremyodle — CHANGES_REQUESTED** [review]
- [x] Change module name to `wco`

File: aws/wco/module.tf [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+
+variable "name" {
+  type        = string
+  default     = "wco"
+  description = "The name of the module"
+}
+
+variable terraform_module {
+  type        = string
+  default     = "gravicore/terraform-gravicore-modules/aws/workspace-cost-optimizer"
```
Comment (jeremyodle) [inline aws/wco/module.tf]: - [x] Change module name to `wco`

**Review by jeremyodle — CHANGES_REQUESTED** [review]

File: .github/labeler.yml [hunk 2]
```diff
@@ -89,4 +89,6 @@ aws/vpc-dhcp:
  - aws/vpc-dhcp/**/*
 aws/vpc-dns:
  - aws/vpc-dns/**/*
- 
+aws/workspace-cost-optimizer:
+ - aws/workspace-cost-optimizer/**/*
```
Comment (jeremyodle) [inline .github/labeler.yml]: - [x] Change module name to `wco`

**Review by jeremyodle — CHANGES_REQUESTED** [review]

File: aws/wco/module.tf (hunk 1 above)
Comment (TysonBeck) [reply aws/wco/module.tf]: Updated name

File: .github/labeler.yml (hunk 2 above)
Comment (TysonBeck) [reply .github/labeler.yml]: Updated labeler

**Review by jeremyodle — APPROVED** [review]

**Pull request merged**